ui/js/values.js

```diff
@@ -114,6 +114,7 @@ treeherder.value("thOptionOrder", {
     "asan": 2,
     "debug": 3,
     "cc": 4,
+    "tsan": 5,
```

Comment:
I think maybe the sheriffs would want these showing next to `asan` (in which case bump the numbers up; they are just used for sort order) - but perhaps worth checking with them.